packages/vuetify/src/directives/touch/index.ts

```diff
@@ -84,7 +96,7 @@ function createHandlers (value: TouchHandlers): TouchStoredHandlers {
   }
 }
 
-function inserted (el: HTMLElement, binding: TouchVNodeDirective, vnode: VNode) {
+function mounted (el: HTMLElement, binding: TouchDirectiveBinding) {
   const value = binding.value!
   const target = value.parent ? el.parentElement : el
   const options = value.options || { passive: true }
```

Comment:
```suggestion
  const options = value.options || passiveEventOptions()
```

Or maybe `const options = passiveEventOptions(value.options)`?

Reply:
IE is dead